rogue-thi-app/components/RoomMap.jsx

```diff
@@ -110,12 +110,10 @@ export default function RoomMap ({ highlight, roomData }) {
 
   async function loadRoomAvailability () {
     const roomAvailabilityData = await getRoomAvailability()
-
     const roomAvailabilityList = Object.fromEntries(Object.entries(roomAvailabilityData).map(([room, openings]) => {
       const availability = openings
         .filter(opening =>
-          new Date(opening.until) > new Date() &&
-          new Date(opening.from) > addSearchDuration(new Date())
```

Comment:
So, I my opinion this should stay because the map shows the availability by color based on the availability for the next hour defined in [`DURATION_PRESET`](https://github.com/neuland-ingolstadt/neuland.app/blob/38f590a56956954399a0d0b764d2bfb2b12c4b1e/rogue-thi-app/lib/backend-utils/rooms-utils.js#L13C14-L13C30). But if, for example the map shows a room as occupied because it's occupied in the next 15 minutes (but at this time still free), the newly added room openings show this short slot which is useless in my opinion. By adding the additional filter, slots under one hour a skipped and the openings reflect the occupancy shown on the map.

Pls share your opinions @pl0ss @Robert27

Reply:
Sometimes I have long breaks between two lessons.
I want to know, if the room of my next lesson is occupied before or not. So I can decide, whether I go early to my next room and do some work there, or I better be there on time, to avoid waiting in the hall.

That is the only reason, why I have added this feature lol. I hate using primuss.